### PR TITLE
Tests for Add and Update Task

### PR DESCRIPTION
**Pull request by bri-personal**
General description of changes:
* Adds Jest test functions for adding and updating a task
* Changes return values of addTask and updateTask to be entire entry for easier testing

Breaking changes: 

* shouldn't be any

Did you run `npm run before-pr`?

- [X] yes
- [ ] no

Does this PR fix/implement an Issue?

- [ ] yes, fixes #XX
- [X] no

Other notes:

### CODE REVIEW

**Review by geoffblay — APPROVED** [review]

File: test/TaskControllers.test.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+            // check that task title is correct
+            await expect(addTask(task)).rejects.toThrowError("duplicate key");
+
+            // tests should be independent, so remove created task
+            await test_manualDeleteTask(testTask.task_id);
+        })
+    })
+
+    describe('updating a task', () => {
```
Comment (MrEminent42) [inline test/TaskControllers.test.ts:57]: Todo: confirm with Davide if this should be in a separate file.

File: test/TaskControllers.test.ts [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        room = await test_manualCreateRoom(property.property_id);
+    })
+
+    afterAll(async () => {
+        // delete temp property cascades to delete room too
+        await test_manualDeleteProperty(property.property_id);
+    })
+
+    describe('adding a task', () => {
+        test('should create a task with a unique task id', async () => {
```
Comment (MrEminent42) [inline test/TaskControllers.test.ts:22]: This does not test if the task id is unique. Rephrase?

File: test/TaskControllers.test.ts [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            // create object test task to add to database
+            // task_id must be specified so we know that function returns correct ID
+            const task: Database['public']['Tables']['Tasks']['Insert'] = {
```
Comment (MrEminent42) [inline test/TaskControllers.test.ts:25]: FYI you don't have to specify a type here. It's useful for reading, though addTask should accept the insert with or without this explicit type.

**Review by MrEminent42 — COMMENTED** [review]
Need to confirm with Davide tomorrow, then we're good to go

**Review by zachstoney30 — APPROVED** [review]

File: test/TaskControllers.test.ts (hunk 2 above)
Comment (MrEminent42) [reply test/TaskControllers.test.ts:22]: Todo in #82

**Pull request merged**